README.md

```diff
@@ -1 +1,26 @@
-# angular-debug-info-readme
\ No newline at end of file
+# Disable debug info
+
+## Overview
+
+You might've noticed all these `ng-scope` and `ng-binding` classes. These are for our test runners to be able to run properly. However, in production, we don't need these. Let's turn them off!
+
+## Objectives
+
+- Describe debug info
+- Disable debug info in production
+
+## Disabling debug info
+
```

Comment:
Add just a little bit more context here - I hadn't really noticed the `ng-binding` classes, so I was caught a bit off guard. Identify what these classes are and what they're doing. Then, go into your explanation below.